en/micro-integrator/docs/develop/creating-unit-test-suite.md

```diff
@@ -172,6 +172,18 @@ Mock services give the opportunity to simulate the actual endpoint.
     -   **Header Value**: Response request header value.
     -   **Send Out Response Payload**: Expected response payload from the service.   
   
+    !!! Note 
+        
+        Please note that the mock service **should have a sub context** with '/' defined, and additional sub contexts should be defined after that.
```

Comment:
Shall we explain why we should do this?

Reply:
According to current implementation, this is how it works. There is no standard to define like this. Should we point the technical limitation here?

Reply:
No we should avoid the technical limitation. I assumed there might be a positive side of defining it like this.